secboot/bootstrap_container_sb.go

```diff
@@ -76,6 +83,35 @@ func (bc *bootstrappedContainer) RemoveBootstrapKey() error {
 	return nil
 }
 
+func (bc *bootstrappedContainer) RegisterKey(primaryKey []byte, unlockKey []byte) {
+	devlinks, err := disksDevlinks(bc.devicePath)
+	if err != nil {
+		logger.Noticef("warning: cannot find symlinks for %s: %v", bc.devicePath, err)
+		return
+	}
+
+	var uuidDevlink string
+	for _, devlink := range devlinks {
+		if strings.HasPrefix(devlink, "/dev/disk/by-uuid/") {
+			uuidDevlink = devlink
+			break
+		}
+	}
+	if uuidDevlink == "" {
+		logger.Noticef("warning: missing by-uuid symlink for %s", bc.devicePath)
```

Comment:
Can you add a small comment that explains why it is okay to skip registering the key instead of returning an error? Also, is it even possible to not have the by-uuid disk dev symlink?

Reply:
I do not think we ever fail when we cannot register keys in the keyring. So this is to keep consistent with secboot.